networkml/__init__.py

```diff
@@ -1,2 +0,0 @@
-import pkg_resources
-__version__ = pkg_resources.get_distribution('networkml').version
```

Comment:
does this conflict with pbr? if not, we should leave this here as it allows users to do the following:

```
import networkml
networkml.__verison__
```

Reply:
It looks like it's deprecated: https://docs.python.org/3/library/importlib.metadata.html

Just reading through pbr.version versus importlib now to see which/either or solution is best.